cBinder/__main__.py

```diff
@@ -6,7 +6,7 @@
     path = os.path.dirname(os.path.dirname(__file__))
     sys.path.insert(0, path)
 
-from cBinder.main import main
+from cBinder.main import main   # noqa: E402
```

Comment:
?

Reply:
This is a configuration to flake8 tool, to ignore checking E402 rule (https://www.flake8rules.com/rules/E402.html) on this line, see http://flake8.pycqa.org/en/2.6.0/config.html